### PR TITLE
First draft of Getting Started

### PR DESCRIPTION
**Pull request by Rajakavitha1**
- Please review @mikenikles 

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/467"></a>

### CODE REVIEW

File: src/routes/docs/beta/getting-started/index.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+You can quickly open a new workspace for the project using a context URL like repository, branch, pull request, issue, or file. Just prefix the URL in the address bar of your browser with `gitpod.io/#`.
+
+For example, try opening [https://gitpod.io/#https://gitlab.com/gitpod/spring-petclinic](https://gitpod.io/#https://gitlab.com/gitpod/spring-petclinic) or [https://gitlab.com/gitpod/spring-petclinic/-/merge_requests/2](https://gitlab.com/gitpod/spring-petclinic/-/merge_requests/2).
+
+[Learn more &rarr;](/docs/context-urls)
+
+## Browser Extension
+
+For convenience, we've made a browser extension that works with Google Chrome and Mozilla Firefox. The extension adds a Gitpod button on every project and branch across GitLab, GitHub, and Bitbucket so you can easily open a new workspace for any existing project.
```
Comment (mikenikles) [inline src/routes/docs/beta/getting-started/index.md:78]: This is a bit outdated in the existing docs. Our extension works with Chromium-based browsers and Firefox. That includes Microsoft Edge, Brave, Chrome and possibly others.

Maybe we use "Chrome-based browser & Firefox"? Have you seen how others refer to these browsers in their docs?

File: src/routes/docs/beta/getting-started/index.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+[Learn more &rarr;](/docs/browser-extension)
+
+## GitLab Integration
+
+GitLab comes with a native Gitpod which is enabled by default for GitLab.com. To use the GitLab integration, you need to enable it from your [user preferences](https://gitlab.com/-/profile/preferences) at GitLab. Then you can choose to open a Gitpod workspace as an alternative to the GitLab Web IDE directly from GitLab for any existing project.
```
Comment (mikenikles) [inline src/routes/docs/beta/getting-started/index.md:84]: "... with a native Gitpod integration which is ..."

File: src/routes/docs/beta/getting-started/index.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+[Learn more &rarr;](/docs/gitlab-integration)
+
+## Example Project
```
Comment (mikenikles) [inline src/routes/docs/beta/getting-started/index.md:88]: 🤔 This seems a bit repetitive given we walk users through an example in detail above. What if we keep this brief and link to a page where we list more examples?

File: src/routes/docs/beta/getting-started/index.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+See for yourself and try opening some of our favorite OSS projects out there:
+
+- [freeCodeCamp](https://gitpod.io/#https://github.com/freeCodeCamp/freeCodeCamp)
+- [Prometheus](https://gitpod.io/#https://github.com/prometheus/prometheus)
+- [GitLab](https://gitpod.io/#https://gitlab.com/gitlab-org/gitlab)
+- [Forem](https://gitpod.io/#https://github.com/forem/forem) powering [dev.to](https://dev.to/)
+- [Nushell](https://gitpod.io/#https://github.com/nushell/nushell)
+
+## Next Steps
```
Comment (mikenikles) [inline src/routes/docs/beta/getting-started/index.md:110]: Great chapter! This is the mental shift that's difficult for many people I've spoken with. The answer to "what do you mean I no longer have a local environment?" is something that's very crucial to get right.

**Review by mikenikles — COMMENTED** [review]
Hey Rajie, thanks for getting that started (no pun intended 🤣). I left some notes / discussion points 💭

File: src/routes/docs/$layout.svelte [hunk 5]
```diff
@@ -24,7 +24,11 @@
 
   let MENU: MenuEntry[];
   $: MENU = isBetaPage
-    ? [M("Getting Started", "beta"), M("🔙 Go back to stable docs", "")]
+    ? [
+        M("Overview", "beta"),
+        M(" Getting Started", "beta/getting-started"),
```
Comment (mikenikles) [inline src/routes/docs/$layout.svelte:29]:
```suggestion
        M(" Getting Started", "beta/getting-started"),
        M("Getting Started", "beta/getting-started"),
```

File: src/routes/docs/beta/quickstart/index.md [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+
+   The revised URL is: `https://gitpod.io/#prebuild/https://github.com/<github_username>/theia`
+
+   Gitpod displays the prebuild progress status with by running the `init`commands in the _.gitpod.yml_ file before you even start a workspace. Later, when you create a new workspace on a branch, or pull/merge request, for which a prebuild exists, the workspace loads much faster, because all dependencies are already downloaded and the code is compiled. For more information, see [prebuilds](/docs/prebuilds).
+
+4. Gitpod starts an automated, and fully configured development environment for the project in a workspace that is now ready for you to code.
+
+## Go example project
+
+The example project that is used in the following section is a [Go](https://github.com/prometheus/prometheus) application. In this task you learn how the Gitpod GitHub App prebuilds a workspace and you can view the prebuild events.
```
Comment (svenefftinge) [inline src/routes/docs/beta/quickstart/index.md:59]: prometheus is not a great starter example. I have created
https://github.com/gitpod-io/golang-cli-starter
let's use that one.

**Review by svenefftinge — COMMENTED** [review]
We need to revise the examples. We currently use very involving complex real world projects. 
Even if I'm a rust user ending up in `nushell` is intimidating. 
We should rather build small starter examples, that have the setup.
I've build something for golang https://github.com/gitpod-io/golang-cli-starter

File: src/routes/docs/beta/quickstart/index.md [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+
+   The revised URL is: `https://gitpod.io/#prebuild/https://github.com/<github_username>/sveltejs-template/`
+
+   Gitpod displays the prebuild progress status with by running the `init`commands in the _.gitpod.yml_ file before you even start a workspace. Later, when you create a new workspace on a branch, or pull/merge request, for which a prebuild exists, the workspace loads much faster, because all dependencies are already downloaded and the code is compiled. For more information, see [prebuilds](/docs/prebuilds).
+
+4. Gitpod starts an automated, and fully configured development environment for the project in a workspace that is now ready for you to code.
+
+## TypeScript example project
+
+The example project that is used in the following section is a [TypeScript](https://github.com/eclipse-theia/theia) application. In this task you learn how the Gitpod GitHub App prebuilds a workspace and you can view the prebuild events.
```
[the remark this answers is not in the document]
Comment (svenefftinge) [reply src/routes/docs/beta/quickstart/index.md:41]: I created https://github.com/gitpod-io/TypeScript-Node-Starter et's use that one.

File: src/routes/docs/beta/quickstart/index.md [hunk 8]
```diff
@@ -0,0 +1,164 @@
...
+
+   The revised URL is: `https://gitpod.io/#prebuild/https://github.com/<github_username>/prometheus/`
+
+   Gitpod displays the prebuild progress status with by running the `init`commands in the _.gitpod.yml_ file before you even start a workspace. Later, when you create a new workspace on a branch, or pull/merge request, for which a prebuild exists, the workspace loads much faster, because all dependencies are already downloaded and the code is compiled. For more information, see [prebuilds](/docs/prebuilds).
+
+4. Gitpod starts an automated, and fully configured development environment for the project in a workspace that is now ready for you to code.
+
+## Rust example project
+
+The example project that is used in the following section is a [Rust](https://github.com/nushell/nushell) application. In this task you learn how the Gitpod GitHub App prebuilds a workspace and you can view the prebuild events.
```
Comment (svenefftinge) [inline src/routes/docs/beta/quickstart/index.md:77]: Let's use https://github.com/gitpod-io/rust-starter instead

**Pull request merged**